tests/vmi_configuration_test.go

```diff
@@ -101,6 +102,7 @@ var _ = Describe("Configurations", func() {
 				}
 				return *vmi.Status.QOSClass
 			}, 10*time.Second, 1*time.Second).Should(Equal(kubev1.PodQOSGuaranteed))
+			Expect(true).To(BeFalse())
```

Comment:
Also this

Reply:
Good eye. Yes, I left them there to test the reporter :)